### PR TITLE
Make img subdir, update RTD tool description, mkdocs toc structure

### PR DESCRIPTION
**Pull request by rpellows**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -1,17 +1,25 @@
...
+    - Developers' Corner:
+        - Contributing: contributing.md
+        - Project Guidelines: guidelines.md
+        - Maintainers' Corner: maintainers_corner.md
+        - Community: community.md
+    - About:
+        - Support: support.md
+        - Legal: legal.md
+        - Community: community.md
+    - ' ': how_to_use.md
```
Comment (vjtirumalai) [inline mkdocs.yml:22]: What does this line do?

**Review by vjtirumalai — COMMENTED** [review]
There are trailing spaces and extra new lines added. Lets keep an eye on them and not add them.

File: mkdocs.yml (hunk 1 above)
Comment (rpellows) [reply mkdocs.yml:22]: Hides the page from the TOC (if you hover you can still see the link).  Mkdocs has no workaround for hiding a page from the TOC.

Comment (rpellows) [reply mkdocs.yml:22]: For the trailing lines, wanted to be consistent and have all files end with newline.  Will look for and remove any trailing spaces.

**Review by vjtirumalai — APPROVED** [review]

**Review by mrmccarthy — COMMENTED** [review]
LGTM

**Pull request merged**